lib/expand.js

```diff
@@ -515,7 +515,20 @@ async function _expandObject({
     }
 
     if(expandedProperty === '@type') {
-      _validateTypeValue(value);
+      // if framing, can be a default object, but need to expand
+      // key to determine that
+      if(_isObject(value)) {
+        const expandedValue = {};
+        for(const [k, v] of Object.entries(value)) {
+          const kk = _expandIri(typeScopedContext, k, {vocab: true});
+          const vv = _asArray(v).map(vvv =>
+            _expandIri(typeScopedContext, vvv, {base: true, vocab: true})
+          );
+          expandedValue[kk] = vv;
+        }
+        value = expandedValue;
```

Comment:
If using `Object.entries`, could also use `Object.fromEntries` to do this transformation:
```suggestion
        value = Object.fromEntries(Object.entries(value).map(([k, v]) => [
          _expandIri(typeScopedContext, k, {vocab: true}),
          _asArray(v).map(vv =>
￼            _expandIri(typeScopedContext, vv, {base: true, vocab: true})
￼          )
        ]));
```

Reply:
It's hard to keep up with the evolving ES6 iterators. This is, indeed, cleaner.